README.md

```diff
@@ -107,9 +107,11 @@ shown bundled together in the Stackdriver Logging UI.
 
 ```javascript
 const lw = require('@google-cloud/logging-winston');
+const winston = require('winston');
```

Comment:
@ofrobots 👋 is this correct?

Reply:
Oh my, so much churn.

Yes, this is correct as per #372. Note that the changes in #372 were overwritten by #371, and this change is undoing that revert.

Reply:
IOW, #371 should never have landed.